src/style.css

```diff
@@ -267,7 +267,7 @@ button:active {
   width: 100%;
 }
 
-tr:nth-last-child(even) {
+tr:nth-child(even) {
```

Comment:
#38 

Problem fixed as choosing elements from bottom to top instead of top to bottom